app/models/mentor/application.rb

```diff
@@ -0,0 +1,118 @@
+module Mentor
+  class Application
+    include ActiveModel::Model
+
+    DB = ActiveRecord::Base.connection
+
+    attr_accessor :id, :team_name
+    attr_accessor :project_id, :project_name
+    attr_accessor :project_plan, :why_selected_project
+    attr_accessor :student0, :student1
+    attr_accessor :first_choice
+
+    def choice
+      first_choice ? 1 : 2
+    end
+
+    def student0=(attrs)
+      attrs     = JSON.parse(attrs).instance_eval(&studentize)
```

Comment:
Interesting, I wrote it like this on purpose, cause I thought it would be **better** readable, since you can basically read it from left to right and have every step visible. To me the other approach looked more inside-out, since you have to wrap in in brackets... But if it looked more familiar to you on the first glance, I can easily change it.

And yes, I went for mutation on purpose. The whole process is basically just mapping database columns to model attributes. So there's no point in allocating memory for the intermediate steps and have their results lying around.